Source/WebKit/GPUProcess/media/RemoteVideoFrameObjectHeap.cpp

```diff
@@ -118,7 +118,8 @@ void RemoteVideoFrameObjectHeap::getVideoFrameBuffer(RemoteVideoFrameReadReferen
     Ref connection = m_connection;
 
     if (videoFrame) {
-        buffer = m_sharedVideoFrameWriter.writeBuffer(videoFrame->pixelBuffer(),
+        RetainPtr<CVPixelBufferRef> pixelBuffer = videoFrame->pixelBuffer();
```

Comment:
Would be nice to use a `protectedPixelBuffer()` getter instead, to avoid a new local variable.

Reply:
Done.